lxd/instance/drivers/driver_qemu.go

```diff
@@ -1224,7 +1224,7 @@ func (d *qemu) start(stateful bool, op *operationlock.InstanceOperation) error {
 	// Copy OVMF settings firmware to nvram file if needed.
 	// This firmware file can be modified by the VM so it must be copied from the defaults.
 	if d.architectureSupportsUEFI(d.architecture) && (!shared.PathExists(d.nvramPath()) || shared.IsTrue(d.localConfig["volatile.apply_nvram"])) {
-		err = d.setupNvram()
+		err = d.setupNvram(true)
```

Comment:
passing true here looks problematic as it will cause an unmount when the function returns, which will potentially undo or conflict with the mount call that happens above on line 1148.

Reply:
We have discussed this point privately with Tom and yes, it looks like an old bug in the existing code. I have fixed that in this PR.